templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -96,11 +96,15 @@ cat >> vars/main.yaml << VARSYAML
 pulp_env: {{ pulp_env | tojson }}
 pulp_settings: {{ pulp_settings | tojson }}
 pulp_scheme: {{ pulp_scheme }}
-{% if python_version == "3.6" %}
-pulp_container_tag: "python36"
+{%- if ci_base_image.count(":") %}
+pulp_default_container: {{ ci_base_image }}
 {% else %}
-pulp_container_tag: "latest"
-{% endif %}
+{%- if python_version == "3.6" %}
+pulp_default_container: {{ ci_base_image + ":python36" }}
```

Comment:
On what platform do we still need to care about Python 3.6?

Reply:
No idea.

Reply:
Let's not do this. Also let's not try to sync the python version on the outside and the inside of the container if we only do it sporadically.

Reply:
I mean, let's drop 3.6.